src/Database/Query/JsonPathHandler.php

```diff
@@ -14,12 +14,6 @@ public function handle(string $path, ?string $alias = null): Expression
     {
         $expression = $this->buildJsonPath($path);
 
-        if ($alias !== null) {
```

Comment:
@rajeshn95 For now comments this out, we need to still discuss how we might want to solve this problem.

Reply:
Ok